aip/general/0122.md

```diff
@@ -108,30 +108,22 @@ A resource ID segment identifies the resource within its parent collection. In
 the resource name `publishers/123/books/les-miserables`, `123` is the resource
 ID for the publisher, and `les-miserables` is the resource ID for the book.
 
-- Resource IDs **may** be either always set by users (required on resource
-  creation), optionally set by users (optional on resource creation,
-  server-generated if unset), or never set by users (not accepted at resource
-  creation). They **should** be immutable once created.
-  - If resource IDs are user-settable, the API **must** document allowed
-    formats. User-settable resource IDs **should** conform to [RFC-1034][];
-    which restricts to letters, numbers, and hyphen, with the first character
-    a letter, the last a letter or a number, and a 63 character maximum.
-    - Additionally, user-settable resource IDs **should** restrict letters to
-      lower-case (`^[a-z]([a-z0-9-]{0,61}[a-z0-9])?$`).
-    - Characters outside of ASCII **should not** be permitted; however, if
-      Unicode characters are necessary, APIs **must** follow guidance in
-      [AIP-210][].
-    - User-settable IDs **should not** be permitted to be a UUID (or any value
-      that syntactically appears to be a UUID).
-  - If resource IDs are not user-settable, the API **should** document the
-    basic format, and any upper boundaries (for example, "at most 63
-    characters").
-  - For more information, see the [create][] standard method.
-
-**Important:** Resources that are declarative-friendly ([AIP-128][]) **must** use
-user-settable resource IDs.
-
-[aip-128]: ./0128.md
+- If resource IDs are user-specified, the API **must** document allowed
```

Comment:
Any guidance on uniqueness? Globally unique? Unique within collection?

Reply:
Is a follow-up PR ok? this one is just to ensure that we *do* introduce new guidance, it's 
not conflicting due to duplicated information.

Reply:
Yep.